android/tangram/jni/platform_android.cpp

```diff
@@ -302,7 +302,10 @@ void featureSelectionCallback(JNIEnv* jniEnv, const std::vector<Tangram::TouchIt
     jresult = reinterpret_cast<jlong>(new std::shared_ptr<Tangram::Properties >(items[0].properties));
     jobject object = jniEnv->NewObject(propertiesClass, propertiesConstructorMID, jresult, true);
 
-    jniEnv->CallVoidMethod(tangramInstance, featureSelectionCbMID, object);
+    jdoubleArray itemCenter = jniEnv->NewDoubleArray(2);
```

Comment:
Not sure about that - array doesn't guarantee any number of values to the receiver. [PointF](https://developer.android.com/reference/android/graphics/PointF.html) is another option.